### ISSUE TITLE
Does anyone reproduce the NWPU-test result?

### ISSUE THREAD

**Issue by wzhiyuan2016** [TencentYoutuResearch/CrowdCounting-P2PNet#12]
![p2p-nwpu](https://user-images.githubusercontent.com/33740622/142800405-9f40c20f-2b06-4f52-b490-5d57b920fb4c.png)

**Comment by lq0104** [comment]
想了解数据预处理这部分是怎么做的？比如训练和测试的分辨率等等
还是训练的超参数具体是什么

**Comment by wzhiyuan2016** [comment]
> 想了解数据预处理这部分是怎么做的？比如训练和测试的分辨率等等 还是训练的超参数具体是什么
我：
输入：512x512 ，其他与shanghaiT A 参数相同，如数据增强（如镜像，随机截取512x512）相同  
测试：整张图输入
loss：相关的计算比例跟作者一样
性能：目前val mae=101.1（训练的不好）
其他设置参数可能需要调，如anchors的数量等

**Comment by lq0104** [comment]
我的输入的crop是128*128，其他与你类似，目前最好的val mae也是100多一点，anchor改成3*3，也这样

**Comment by wzhiyuan2016** [comment]
> 我的输入的crop是128_128，其他与你类似，目前最好的val mae也是100多一点，anchor改成3_3，也这样

可能是一些细节论文并未提到。

**Comment by swzhang5** [comment]
> > 我的输入的crop是128_128，其他与你类似，目前最好的val mae也是100多一点，anchor改成3_3，也这样
> 
> 可能是一些细节论文并未提到。

请问你现在复现出来了吗？我也遇到了复现NWPU的问题，也许可以交流一下

**Comment by msn199959** [comment]
> 

我也有相同的问题，或许可以交流一下 msn199959@163.com

**Comment by David-rn** [comment]
@msn199959 and @swzhang5 were you able to reproduce the results of the paper in the datasets?

**Comment by swzhang5** [comment]
> @msn199959 and @swzhang5 were you able to reproduce the results of the paper in the datasets?

No, tried many tricks, but there still some gap between my results and the results in the paper.
